src/components/media-views.js

```diff
@@ -8,6 +8,11 @@ import { proxiedUrlFor } from "../utils/media-url-utils";
 import { buildAbsoluteURL } from "url-toolkit";
 import { SOUND_CAMERA_TOOL_TOOK_SNAPSHOT } from "../systems/sound-effects-system";
 import { promisifyWorker } from "../utils/promisify-worker.js";
+import pdfjs from "pdfjs-dist";
+
+// Using external CDN to reduce build size
+pdfjs.GlobalWorkerOptions.workerSrc =
```

Comment:
In the interest of self hosting, might be better to point at one of the general cdn versions of this instead https://github.com/mozilla/pdf.js#including-via-a-cdn